theme/less/gouvfr/site-search.less

```diff
@@ -17,4 +17,8 @@
             color: white;
         }
     }
+
+    .home & .form-group {
```

Comment:
This is not the same. It means exactly: `.home .site-search .form-group`. The `&` in less (or sass) replace the current entire selector.